api/capi/src/nntrainer.cpp

```diff
@@ -77,6 +77,30 @@ int ml_nnmodel_construct(ml_nnmodel_h *model) {
   return status;
 }
 
+int ml_nnmodel_construct_with_conf(const char *model_conf, ml_nnmodel_h *model) {
+  int status = ML_ERROR_NONE;
+  ml_nnmodel *nnmodel;
+
+  std::ifstream conf_file(model_conf);
+  if (!conf_file.good()) {
+    std::cerr << "Error: Cannot open model configuration file : " << model_conf << std::endl;
+    return ML_ERROR_INVALID_PARAMETER;
+  }
+  if (!model) {
```

Comment:
I think this should be `if(model)`

Reply:
My mistake. This should be removed. Thanks.